docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -0,0 +1,411 @@
+----
+
+. Confirm the mon ip address is present on the source node `os-net-config`
+configuration located in `/etc/os-net-config`:
++
+----
+
+[tripleo-admin@controller-0 ~]$ grep "172.17.3.60" /etc/os-net-config/config.yaml
+    - ip_netmask: 172.17.3.60/24
+----
+
+. Edit the config file `/etc/os-net-config/config.yaml` and remove the
+`ip_netmask` line retrieved in the previous step.
+
+. Save the file and refresh the node network configuration with the following
+command:
++
+----
+sudo os-net-config -c /etc/os-net-config/config.yaml
+----
+
+. Verify, using the `ip` command, that the IP address is not present in the source
+node anymore.
+
+ssh into the target node, for example `cephstorage-0`, and add the ip address
+that will be bound to the new mon.
+
+. On the target node, edit the config file `/etc/os-net-config/config.yaml` and
+add the `- ip_netmask: 172.17.3.60` line removed in the source node.
+
+. Save the file and refresh the node network configuration with the following
+command:
++
+----
+sudo os-net-config -c /etc/os-net-config/config.yaml
+----
+
+. Verify, using the `ip` command, that the IP address is present in the target
+node.
+
+Get the Ceph spec and set the mon daemons to `unmanaged`:
+
+. Get the ceph mon spec:
++
+----
+ceph orch ls --export mon > mon.yaml
+----
+
+.Edit the retrieved spec and add the `unamanged: true` keyword:
++
+[source,yaml]
+----
+service_type: mon
+service_id: mon
+placement:
+  label: mon
+unamanged: true
+----
+
+. Save the spec in /tmp/mon.yaml
+. Apply the spec with cephadm using the orchestrator:
++
+----
+sudo cephadm shell -m /tmp/mon.yaml
+ceph orch apply -i /mnt/mon.yaml
+----
+
+The mon daemons are marked as <unmanaged>, and it's now possible to redeploy
+the existing daemon and bind it to the migrated ip address.
+
+. Delete the existing mon on the target node:
++
+----
+$ ceph orch daemon add rm mon.<target_node> --force
+----
+
+. Redeploy the new mon on the target using the old IP address:
++
+----
+$ ceph orch daemon add mon <target_node>:<ip address>
+----
+
+* Replace <target_node> with the hostname of the target node enrolled in the Ceph
+cluster
+* Replace <ip address> with the ip address of the migrated address
+
+Get the Ceph spec and set the mon daemons to `unmanaged: false`:
+
+. Get the ceph mon spec:
++
+----
+ceph orch ls --export mon > mon.yaml
+----
+
+.Edit the retrieved spec and set the `unamanged` keyword to `false`:
++
+[source,yaml]
+----
+service_type: mon
+service_id: mon
+placement:
+  label: mon
+unamanged: false
+----
+
+. Save the spec in /tmp/mon.yaml
+. Apply the spec with cephadm using the orchestrator:
++
+----
+sudo cephadm shell -m /tmp/mon.yaml
+ceph orch apply -i /mnt/mon.yaml
+----
+
+The new mon runs on the target node with the original IP address.
+As last step of the mon migration, you need to refresh the cephadm information
+and reconfigure the existing daemons to exchange the map with the updated mon
+references.
+
```

Comment:
Are we eventually going to split this procedure into multiple procedures? If not, we don't need this "introductory" text.

Reply:
Ack, this clarifies the next steps, but I'm ok to not have this amount of details.

Reply:
Removed